tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>Over all looks really good, clear and to the point.</p><p>Well done!</p><p> </p><p>Let me know if you have questions, or if you want further clarifications for any of my notes.</p><p> </p><p>If you don't agree with any of the notes it perfectly fine and we should discuss it, or just keep it as a question for the review with Eden.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>